packages/core/test/scuttle.spec.js

```diff
@@ -18,9 +17,82 @@ const getPropsGroups = (g) => ({
         Object.getOwnPropertyDescriptor(g, p).writable
     )
     // remove hardcoded props scuttling ignores on purpose
-    .filter((p) => !['Compartment', 'Error', 'globalThis'].includes(p)),
+    .filter((p) => !avoidForLavaMoatCompatibility.includes(p)),
 })
 
+const {scuttle, avoidForLavaMoatCompatibility} = (function(){
+  function fakeCompartment() {
```

Comment:
Why doesn't this use a real compartment?

Reply:
Currently worried loading SES just for a compartment might be an overkill, plus it's just a test, so whether the list of intrinsics is even accurate or not doesn't matter that much. Perhaps we can revisit in the future

Reply:
We did a thought experiment and it was awful.